src/views/AccessDenied/AccessDenied.js

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import React from 'react';
+import { withRouter } from 'react-router-dom';
+import { Grid, Button, Typography, Select, FormControl, MenuItem, Paper } from '@material-ui/core';
+import { Auth } from '@cosmotech/core';
+import { Trans, useTranslation } from 'react-i18next';
+import { TranslationUtils } from '../../utils';
+import useStyles from './style';
+import Countdown from 'react-countdown';
+
+const AccessDenied = (props) => {
+  const classes = useStyles();
+  const { t, i18n } = useTranslation();
+  const year = new Date().getFullYear();
+
+  const DELAY_TO_TIMEOUT = 30;
+
+  const timeoutRenderer = ({ seconds }) => {
+    return (
+      <Typography className={classes.timeout}>
+        {t('views.accessdenied.signouttimeout', 'You will be automatically signed out in {{seconds}}... seconds', {
+          seconds: seconds,
+        })}
+      </Typography>
+    );
+  };
+
+  return (
+    <div className={classes.root}>
+      <Grid className={classes.grid} container>
+        <Grid className={classes.quoteContainer} item lg={5}>
+          <div className={classes.quote}>
+            <div className={classes.quoteInner}>
+              <Typography className={classes.quoteText} variant="h1">
+                {t('views.signin.title', 'Azure Sample Web Application')}
+              </Typography>
+            </div>
+          </div>
+        </Grid>
+        <Grid className={classes.content} item lg={7} xs={12} container>
+          <div className={classes.content}>
+            <div className={classes.contentHeader}>
+              <form className={classes.form}>
```

Comment:
Do we really need a form here ? There is no `input` to send, we only have a button to force log out

Reply:
It is a graphic build from sign in page. I have tryed to simplify it but each time it break the design.